src/main/java/ua/softserveinc/tc/controller/admin/DayOffController.java

```diff
@@ -64,6 +68,7 @@
         if (dayOffService.dayOffExist(dayOff.getName(), dayOff.getStartDate())) {
             log.warn("There is another day off with the same name: " + dayOff.getName() + ", or" +
                     "with the same start date: " + dayOff.getStartDate());
+            return new ResponseEntity<>(HttpStatus.NOT_FOUND);
```

Comment:
Try think again)))
This method doesn't find the same dayoffs! It find dayOff that OR has the same name, OR has the same start Time. This is absolutely different things. And it has bad impact on project.